gatsby/src/components/SEO/SEO.tsx

```diff
@@ -65,6 +70,7 @@ export const SEO: React.FC<SEOProps> = ({ title, description }: SEOProps) => (
 			},
 		]}
 	>
+		{noIndex && <meta name="robots" content="noindex" />}
```

Comment:
We could probably have a test directly on this component that the noindex tag gets added

Reply:
Done